app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileResetPinActivityTest.kt

```diff
@@ -803,7 +803,7 @@ class ProfileResetPinActivityTest {
   }
 
   @Test
-  fun testResetPin_inputPinFields_saveButtonIsClickable_clearPinField_saveButtonIsNotClickable() {
+  fun testProfileResetPin_inputPinField_clickableSaveButton_clearPinField_notClickableSaveButton() {
```

Comment:
testProfileResetPin_inputPinField_clickableSaveButton_clearPinField_saveButtonIsNotClickable

Reply:
Exceeds the limit, changed to notClickableSaveButton throughout the file.

Reply:
testProfileResetPin_inputPin_clickableSaveButton_clearPin_saveButtonIsNotClickable

Let's remove the `Field` from everywhere.

Reply:
Removed Fields throughout the program except at line 468: 
`  fun testProfileResetPin_inputPin_changeConfig_inputFieldsExist_saveButtonIsClickable() ` where I feel it is needed. Maybe we could replace it with `pinAndConfirmPinExist` ?